packages/payment-detection/diagram.svg

```diff
@@ -0,0 +1,258 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
this is autogenerated from the diagram.yuml file, using https://github.com/jaime-olivares/yuml-diagram or the yuml extension